tasm-staff-editor/src/components/EditorView/ExhibitContent.js

```diff
@@ -5,13 +5,13 @@ export default function ExhibitContent({ content, onChange }) {
   return (
     <div>
       <label htmlFor="content" className="text-white">Content:</label>
-      <div className="mt-2">
+      <div className="mt-1">
         <textarea
           id="content"
           name="content"
           onChange={onChange}
           rows={3}
-          className="block mb-6 w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-lightBlue-600 sm:leading-6"
+          className="mb-6 w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-lightBlue-600 sm:leading-6"
```

Comment:
I would still like to turn these long className strings into css classes or some other method of shortening them.
Could be out of scope for this PR though